app/controllers/PathManagerController.scala

```diff
@@ -81,4 +94,6 @@ object PathManagerController extends Controller {
     Ok(s"current id = $currentId")
   }
 
+  def argoOk(json: JsValue) = Ok(Json.obj("data" -> json)).as("application/vnd.argo+json")
```

Comment:
What is this crazyness? Google returns a [single response for this mimetype](https://www.google.co.uk/?gws_rd=ssl#q=%22application%2Fvnd.argo%22)